setup.py

```diff
@@ -47,32 +46,37 @@
     + extras_require["dev"]
 )
 
-with open("./README.md") as readme:
+readme_path, readme_content_type = "./README.rst", "text/x-rst"
+if os.path.exists("./README.md"):
+    readme_path, readme_content_type = "./README.md", "text/markdown"
+
+with open(readme_path) as readme:
     long_description = readme.read()
 
 
 setup(
-    name="<PYPI_NAME>",
+    name="ape-hardhat",
     use_scm_version=True,
     setup_requires=["setuptools_scm"],
-    description="""<PYPI_NAME>: <SHORT_DESCRIPTION>""",
+    description="""ape-hardhat: Ape network provider for Hardhat""",
     long_description=long_description,
-    long_description_content_type="text/markdown",
+    long_description_content_type=readme_content_type,
     author="ApeWorX Ltd.",
     author_email="admin@apeworx.io",
-    url="https://github.com/ApeWorX/<REPO_NAME>",
+    url="https://github.com/ApeWorX/ape-hardhat",
     include_package_data=True,
     install_requires=[
+        "eth-ape>=0.1.0a22",
```

Comment:
```suggestion
        "eth-ape==0.1.0a23",
```

I highly recommend pinning the version here until.
There will likely be many breaking changes with core ape until we are satisfied and reach 1.0.0